src/main/java/tuteez/logic/commands/DeleteCommand.java

```diff
@@ -18,11 +18,13 @@
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
+    public static final String COMMAND_WORD_ALT = "del";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " | " + COMMAND_WORD_ALT
```

Comment:
Could "|" be ambiguous? I know some languages use that as the "or" symbol so programmers can easily understand. But the layman users this might not be immediately clear. Lets discuss this during our next huddle if necessary 👍

Reply:
Yes, I also considered this. My worry is that using "or", "OR" or "/" could get mixed up with the existing commands. Let's discuss this again

Reply:
I think "or" might be counter-intuitive